README.md

```diff
@@ -127,6 +127,53 @@ spec:
         hostPath:
           path: /etc/localtime
 ```
+
+For Kubernetes >= 1.9 you need to add selector tag as bellow:
+
+```yaml
+apiVersion: extensions/v1beta1
```

Comment:
Another nit:

To make the README.md more short and simple, i suggest we change `Start DaemonSet` section to:
```
* Edit [node-problem-detector.yaml](https://github.com/kubernetes/node-problem-detector/blob/master/node-problem-detector.yaml) to fit your environment: Set `log` volume to your system log directory. (Used by SystemLogMonitor)
* Create the DaemonSet with `kubectl create -f node-problem-detector.yaml`
* If needed, you can use [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/)
to overwrite the `config/`.
```

Reply:
Agreed. Do I open new pr?

Reply:
I'm just wondering if we can have link a link to kubernetes deployment >=1.9 and one another bellow to k8s 1.9. Just to be more clear one what's need to be modified between this major change introduce in 1.9. User have just to copy/paste instead of finding what's to change in the yaml. Are you agree?

Reply:
As 1.10 is about to be released, i think it is fine to bump DaemonSet apiVersion to `apps/v1` and make this only workable for Kubernetes above 1.9.